pixeltable/exprs/column_property_ref.py

```diff
@@ -87,14 +94,17 @@ def eval(self, data_row: DataRow, row_builder: RowBuilder) -> None:
             assert data_row.has_val[self._col_ref.slot_idx]
             data_row[self.slot_idx] = data_row.file_paths[self._col_ref.slot_idx]
             return
-        elif self.is_error_prop():
+        elif self.is_cellmd_prop():
             exc = data_row.get_exc(self._col_ref.slot_idx)
             if exc is None:
                 data_row[self.slot_idx] = None
             elif self.prop == self.Property.ERRORTYPE:
                 data_row[self.slot_idx] = type(exc).__name__
-            else:
+            elif self.prop == self.Property.ERRORMSG:
                 data_row[self.slot_idx] = str(exc)
+            else:
+                data_row[self.slot_idx] = self.create_cellmd_exc(exc)
```

Comment:
Suggestion: add `assert self.prop == self.Property.CELLMD` for clarity.

This implementation will need to change when we add non-error cell md.